Specs/Scene/GroundPrimitiveSpec.js

```diff
@@ -0,0 +1,735 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/GroundPrimitive',
+        'Core/Cartesian3',
+        'Core/ColorGeometryInstanceAttribute',
+        'Core/ComponentDatatype',
+        'Core/destroyObject',
+        'Core/Ellipsoid',
+        'Core/Geometry',
+        'Core/GeometryAttribute',
+        'Core/GeometryInstance',
+        'Core/GeometryInstanceAttribute',
+        'Core/PolygonGeometry',
+        'Core/PrimitiveType',
+        'Core/Rectangle',
+        'Core/RectangleGeometry',
+        'Core/RuntimeError',
+        'Core/ShowGeometryInstanceAttribute',
+        'Core/Transforms',
+        'Renderer/ClearCommand',
+        'Scene/MaterialAppearance',
+        'Scene/OrthographicFrustum',
+        'Scene/Pass',
+        'Scene/PerInstanceColorAppearance',
+        'Scene/Primitive',
+        'Scene/PrimitiveCollection',
+        'Scene/SceneMode',
+        'Specs/BadGeometry',
+        'Specs/createContext',
+        'Specs/createFrameState',
+        'Specs/createScene',
+        'Specs/pick',
+        'Specs/pollToPromise',
+        'Specs/render'
+    ], function(
+        GroundPrimitive,
+        Cartesian3,
+        ColorGeometryInstanceAttribute,
+        ComponentDatatype,
+        destroyObject,
+        Ellipsoid,
+        Geometry,
+        GeometryAttribute,
+        GeometryInstance,
+        GeometryInstanceAttribute,
+        PolygonGeometry,
+        PrimitiveType,
+        Rectangle,
+        RectangleGeometry,
+        RuntimeError,
+        ShowGeometryInstanceAttribute,
+        Transforms,
+        ClearCommand,
+        MaterialAppearance,
+        OrthographicFrustum,
+        Pass,
+        PerInstanceColorAppearance,
+        Primitive,
+        PrimitiveCollection,
+        SceneMode,
+        BadGeometry,
+        createContext,
+        createFrameState,
+        createScene,
+        pick,
+        pollToPromise,
+        render) {
+    "use strict";
+    /*global jasmine,describe,xdescribe,it,xit,expect,beforeEach,afterEach,beforeAll,afterAll,spyOn,fail*/
+
+    var context;
+    var frameState;
+    var us;
+
+    var ellipsoid;
+    var rectangle;
+
+    var depthColor;
+    var rectColor;
+
+    var rectangleInstance;
+    var depthPrimitive;
+
+    beforeAll(function() {
+        context = createContext();
```

Comment:
Are you sure these tests need to use the context directly?  The trend is to use a scene instead, #1259.